Sources/ScipioKit/SwiftPM+Compatibility.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import TSCBasic
+import Basics
+import PackageModel
+
+// TODO: Remove TSCBasics
+// Since Swift 5.10, SwiftPM removes swift-tools-support-core(TSC)
+// So all interfaces are replaced from TSCBasics.AbsolutePath to Basics.AbsolutePath
```

Comment:
```suggestion
// Since Swift 5.10, SwiftPM removes swift-tools-support-core(TSC), 
// so all interfaces are replaced from TSCBasics.AbsolutePath to Basics.AbsolutePath.
```